inyoka_theme_default/templates/confirm_action_flash.html

```diff
@@ -0,0 +1,17 @@
+{#
+  confirm_action_flash.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  :copyright: (c) 2011-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+<div>
```

Comment:
afaik it its impossible to extend inside this one. However, macros for the form could be used. So if you want to use them, try it. ;)